week-3/F-array-forEach/exercise.js

```diff
@@ -9,7 +9,22 @@
 */
 
 // Complete this function -->
-function fizzBuzz() {}
+var numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+function fizzBuzz(number) {
+  if (number % 3 === 0) {
+    return "Fizz";
+  }
+  if (number % 5 === 0) {
+    return "Buzz";
+  }
+  if (number % 3 === 0 && number % 5 === 0) {
+    return "FizzBuzz";
+  }
+  return number;
+}
+numbers.map(fizzBuzz).forEach(function(number) {
+  console.log(number);
+});
```

Comment:
This is good - you have added a number which tests the missing case. But the output isn't correct for the number 15. Can you see why :slightly_smiling_face:  ?